src/components/_shared/title.js

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 
 import styles from './title.module.scss';
 
-const Title = ({ title, section }) => {
+const Title = ({ title, section, teamColor }) => {
 
-    return <div className={section === "games" ? styles.gameStats : styles.totalStats}>{title}</div>;
+    return <div style={{backgroundColor: `${teamColor}`}} className={section === "games" ? styles.gameStats : styles.totalStats}>{title}</div>;
```

Comment:
No need for string literals; needs spaces around object:

```
style={{ backgroundColor: teamColor }} 
```

Also let's use single quotes for javascript, `"games"` should be `'games'`

Reply:
Cool, I went through all the js files and changed double quotes to single.